spec/features/replies/edit_spec.rb

```diff
@@ -34,7 +34,7 @@ def find_reply_on_page(reply)
     expect(page).to have_no_selector('.post-container')
     within('#post-editor') do
       fill_in 'reply_content', with: 'other text'
-      click_button 'Save'
+      click_button 'submit_button'
```

Comment:
'Save' is actually clearer as to what the button is doing, did something go wrong with that?

Reply:
We now have two save buttons on the page, Save-but-save-the-reply and Save-but-save-the-post-note (just hidden) so this was raising AmbiguousMatch. I did not see a helpful container element that I could use to avoid this problem with a `within`, so I used the CSS id which was already set. I could see about changing the existing CSS id to something slightly less opaque instead if that works better?

Reply:
That makes sense! And yeah a less opaque id might be good if that's not too much extra work.